subconscious/column.py

```diff
@@ -0,0 +1,18 @@
+from .model import Column
```

Comment:
In fact I wanted to move Column to column.py , but thought a bigger refactor in the coordinator project, may be I'll just do that